docs/user/metrics/timespan.md

```diff
@@ -288,6 +323,12 @@ TODO. To be implemented in [bug 1648442](https://bugzilla.mozilla.org/show_bug.c
 
 </div>
 
+<div data-lang="Rust" class="tab">
+
+The raw API is not supported in Rust.
```

Comment:
cc @Dexterp37 -- I explicitly took out this API. I'm not sure we really need/should provide it on the Rust/FOG side right now. What do you think?

Reply:
I agree on the "right now part", but we also know this covers the know use-case of using external clocks. We definitely know that sync will need it, for example. I'm fine with filing a bug for adding it later. I'd recommend reducing the amount of binding-specific APIs (or lack of APIs!) to not confuse consumers :)

Reply:
```suggestion
The raw API is not supported in Rust. See [bug 1680225](https://bugzilla.mozilla.org/show_bug.cgi?id=1680225).
```